website/pages/guides/integrations/with-gatsby.mdx

```diff
@@ -5,6 +5,14 @@ tags: ["gatsby", "websites", "ssr"]
 author: KenzoBenzo
 ---
 
+# Migration Notes
+
+## Changes
+
+- The Gatsby plugin has been renamed from `gatsby-plugin-chakra-ui` to
+  `@chakra-ui/gatsby-plugin`. Please make sure to have updated this when
+  installing ChakraUI in your next Gatsby project.
```

Comment:
Same as aboce, can you add a space so it's `Chakra UI`?

Reply:
resolved in latest push :)